domain-models/samples/AU128.gas.ttl

```diff
@@ -1,42 +1,34 @@
 @prefix dct: <http://purl.org/dc/terms/> .
 @prefix foaf: <http://xmlns.com/foaf/0.1/> .
 @prefix foiType: <http://pid.geoscience.gov.au/def/voc/featureofinteresttype/> .
+@prefix gas: <http://pid.geoscience.gov.au/def/ga-samples> .
 @prefix geo: <http://www.opengis.net/ont/geosparql#> .
 @prefix org: <http://www.w3.org/ns/org#> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix samfl: <http://def.seegrid.csiro.au/ontology/om/sam-lite#> .
 
 <http://pid.geoscience.gov.au/org/ga/geoscienceaustralia> a org:Organization ;
     foaf:name "Geoscience Australia" .
 
 <http://pid.geoscience.gov.au/sample/AU128> 
-    a samfl:Specimen ;
+    a gas:Specimen ;
     rdfs:label "Sample igsn:AU128" ;
-    samfl:currentLocation [ 
-        a samfl:Location ;
+    gas:currentLocation [ 
+        a dct:Location ;
         dct:description "GA Services building" ;
     ] ;
-    samfl:materialClass <http://vocabulary.odm2.org/medium/rock> ;
-    samfl:sampledFeature <http://pid.geoscience.gov.au/site/17594> ;
-    samfl:samplingLocation [ 
-        a samfl:Point ;
+    gas:hasSampledFeature <http://pid.geoscience.gov.au/site/17594> ;
```

Comment:
Like with the change to GeoSPARQL below, let's replace this one with `sosa:isSampleOf` since it's a 1:1. It's only the other property chain axioms of `gas:samplingMethod` etc we shouldn't change.